docker-compose.yml

```diff
@@ -88,19 +105,26 @@ services:
       - .:/ProtectedPlanet
       - protectedplanet_node_modules:/ProtectedPlanet/node_modules
       - protectedplanet_bundler:/usr/local/bundle
-    network_mode: host
+    ports:
+      - "3035:3035"
+    networks:
+      - protectedplanet
 
   api:
     container_name: protectedplanet-api
     build:
       context: ${API_PATH}
-    command: /bin/bash -l -c "bundle exec rackup"
+    command: /bin/bash -l -c "bundle exec rackup --host 0.0.0.0"
     volumes:
       - ${API_PATH}:/ProtectedPlanetApi
-      - ${SSH_AUTH_SOCK}:${SSH_AUTH_SOCK}
-    network_mode: host
+      # - /run/host-services/ssh-auth.sock:/ssh-agent # Use this setting for Mac OS X and comment out ${SSH_AUTH_SOCK}:/ssh-agent below
+      - ${SSH_AUTH_SOCK}:/ssh-agent
+    ports:
+      - "9292:9292"
```

Comment:
Please change to "9292" (without host port), so it won't conflict with host port. This is Jenkins test, no one will connect from the host.